lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/ICloudBlobClient.cs

```diff
@@ -56,9 +56,9 @@ internal interface ICloudBlobClient
         ICloudBlockBlob GetBlockBlobReference(Uri uri);
 
         /// <summary>
-        /// Paginates a blob listing with prefix.
+        /// Paginates a blob listing with container and directory.
         /// </summary>
-        BlobResultSegment ListBlobsSegmented(string prefix, bool useFlatListing, BlobListingDetails blobListingDetails, int? maxResults, 
+        BlobResultSegment ListBlobsSegmented(string containerName, string directoryName, bool useFlatListing, BlobListingDetails blobListingDetails, int? maxResults, 
```

Comment:
"directoryName" is confusing since this could actually be a path. I suggest using "relativeAddress" to match the blob API.

Reply:
Done.